storage/onpremise/mongodb/outputs.tf

```diff
@@ -43,3 +43,37 @@ output "unused_variables" {
     "validity_period_hours" = var.validity_period_hours
   }
 }
+
+#new Outputs 
+output "env" {
+  description = "Elements to be set as environment variables"
+  value = ({
+    "Components__TableStorage"  = var.adapter_class_name
+    "MongoDB__Host"             = local.mongodb_dns
+    "MongoDB__Port"             = "27017"
+    "MongoDB__Tls"              = var.tls
+    "MongoDB__ReplicaSet"       = var.replicaset
+    "MongoDB__DatabaseName"     = var.mongodb.databases_names[0]
+    "MongoDB__DirectConnection" = var.direct_connection
+    "MongoDB__CAFile"           = "${var.path}/chain.pem"
+  })
+
+}
+
+output "env_secret" {
+  description = "Secrets to be set as environment variables"
+  value = [
+    kubernetes_secret.mongodb_user.metadata[0].name
+  ]
+}
```

Comment:
Why not take `mongodb_user_credentials `secret if it has the correct keys ?

Reply:
In this case I don't understand that "correct keys" thing you mentionned in this comment :

https://github.com/aneoconsulting/ArmoniK.Infra/pull/152#discussion_r1686409725